app/services/SubscriptionStatusService.scala

```diff
@@ -34,7 +34,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class SubscriptionStatusService @Inject()(val appConfig: AppConfig,
                                           businessDetailsConnector: BusinessDetailsConnector,
```

Comment:
BuisnessDetailsConnector should no longer be used, ensure you have a look through and delete anything which is now redundant from it's removal such as:
- Models
- HttpParsers
- Tests
- etc